### PR TITLE
inital ops infra

### PR DESCRIPTION
**Pull request by david-wirelab**

### CODE REVIEW

File: instances.tf [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+resource "aws_instance" "BastionHostLinux" {
+  ami                    = "${data.aws_ami.linux_connectivity_tester.id}"
+  instance_type          = "${var.instance_type}"
+  subnet_id              = "${aws_subnet.OPSSubnet.id}"
+  vpc_security_group_ids = ["${aws_security_group.Bastions.id}"]
+
+  tags {
+    Name = "${local.name_prefix}ec2-linux"
```
Comment (nick-otter) [inline instances.tf]: Additional tags required, as per dq routing doc:
computed values should result in the below:

```hcl
  tags {
    Name             = "bastionlinux01-dq-bastions-dq-ops-prod"
    Service          = "dq-bastions"
    EnvironmentGroup = "dq-ops"
    Environment      = "prod"
  }
```

File: instances.tf [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  user_data = "CHECK_self=127.0.0.1:80 CHECK_google=google.com:80 CHECK_googletls=google.com:443 LISTEN_http=0.0.0.0:80"
+}
+
+resource "aws_security_group" "Bastions" {
+  vpc_id = "${aws_vpc.opsvpc.id}"
+
+  tags {
+    Name = "${local.name_prefix}sg"
```
Comment (nick-otter) [inline instances.tf]: Additional tags required, as per dq routing doc:
something like this:
```hcl
  tags {
    Name             = "sg-dq-bastions-dq-ops-prod"
    Service          = "dq-bastions"
    EnvironmentGroup = "dq-ops"
    Environment      = "prod"
  }
```

File: tests/ops_test.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            module "ops" {
+              source = "./mymodule"
+
+              providers = {
+                aws = "aws"
+              }
+
+              cidr_block            = "10.2.0.0/16"
+              vpc_subnet_cidr_block = "10.2.1.0/24"
+              az                    = "eu-west-2a"
```
Comment (nick-otter) [inline tests/ops_test.py]: need to specify in the environment zone etc so that these can be used in the tags

File: vpc.tf [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+locals {
+  name_prefix = "${var.name_prefix}ops-"
+}
+
+resource "aws_vpc" "opsvpc" {
+  cidr_block = "${var.cidr_block}"
+
+  tags {
+    Name = "${local.name_prefix}vpc"
```
Comment (nick-otter) [inline vpc.tf]: see naming scheme, similar to above

File: vpc.tf [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  }
+}
+
+resource "aws_internet_gateway" "OpsRouteToInternet" {
+  vpc_id = "${aws_vpc.opsvpc.id}"
+
+  tags {
+    Name = "${local.name_prefix}igw"
```
Comment (nick-otter) [inline vpc.tf]: see naming scheme, similar to above

File: vpc.tf [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+resource "aws_subnet" "OPSSubnet" {
+  vpc_id                  = "${aws_vpc.opsvpc.id}"
+  cidr_block              = "${var.vpc_subnet_cidr_block}"
+  map_public_ip_on_launch = true
+  availability_zone       = "${var.az}"
+
+  tags {
+    Name = "${local.name_prefix}subnet"
```
Comment (nick-otter) [inline vpc.tf]: see naming scheme, similar to above

File: instances.tf (hunk 1 above)
Comment (chrisns) [reply instances.tf]: in hindsight I think we should revisit naming and tagging post ITHC. will update the board accordingly

**Pull request merged**